<launch>
  <arg name="kiss_icp_matcher_param_path" default="$(find-pkg-share lidar_scan_matcher)/config/kiss_icp_matcher.param.yaml"/>

  <node pkg="lidar_scan_matcher" exec="kiss_icp_matcher_node" name="kiss_icp_matcher_node" output="screen">
    <remap from="points_raw" to="filtered_points"/>
    <param from="$(var kiss_icp_matcher_param_path)"/>
  </node>

</launch>
